pkg/webconfig/webconfig.go

```diff
@@ -60,6 +63,44 @@ type OverrideProperties struct {
 	NginxServesStaticFiles bool
 }
 
+func (overrides *OverrideProperties) FromEnvironment(ctx *gcp.Context) error {
```

Comment:
nit: I would call this `UpdateFromEnvironment` to be explicit that this is mutating the object rather than just returning a value.

Reply:
Good call!